### PR TITLE
Feature/13990 UI automation for pb is13721and13019

### PR DESCRIPTION
**Pull request by Shital15683**
Hi All,
This PR is consist of UI Automation of sprint 3 PBIs below:
13721,13019,13023 and 13024.
Kindly review.
Thanks.

### CODE REVIEW

File: PageObjects/essui-encselectionpageObjects.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "encselectionpageObjectsConfig": 
+    {        
+        "encNameSelector":"text=ENC name",
+        "startLinkSelector":".linkStartAgain",
+        "textAboveTableSelector":"text=Select up to 100 ENCs and make an exchange set",
+        "textAboveTable":"Select up to 100 ENCs and make an exchange set",  
+        "rightTableRowsCountSelector":"//div/div[2]/div[3]/div[1]/table/tbody/tr",
+        "rightTableEncPositionSeletor":"//div/div[2]/div[3]/div[1]/table/tbody/tr[1]/td[1]",
+        "firstCheckBoxSelector":"[aria-label='AU220150'] input[type='checkbox']",
```
Comment (RaviK2233) [inline PageObjects/essui-encselectionpageObjects.json:10]: can we please change this locator to generic. This currently has enc number. should we change the test data in future, the test will fail as locator is not valid.

Comment (Shital15683) [reply PageObjects/essui-encselectionpageObjects.json]: Hello Ravi,
Added generic locator. please Review.

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Please resolve merge conflicts

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Sorry Kishor a recent update to main has created further merge conflicts - please resolve the conflicts

**Pull request merged**